target/linux/ath79/dts/qca9558_engenius_emr3000.dts

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	model = "EnGenius EMR3000";
+	compatible = "engenius,emr3000", "qca,qca9558";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "emr3000:white:power";
+			gpios = <&gpio 18 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		offline {
+			label = "emr3000:amber:offline";
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+		};
+
+		wps {
+			label = "emr3000:orange:wps";
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+		};
+
+		mesh {
+			label = "emr3000:blue:mesh";
+			gpios = <&gpio 19 GPIO_ACTIVE_HIGH>;
+		};
```

Comment:
Hmm, vendor page tells me the following:

LED Indicators
1 x 4 Color-Status LED –
White: Mesh & Internet Connection Status
Blue: Bluetooth Connection Status
Red: No Internet Connection or No Mesh Connection
Orange: Mesh Connection Signal is Weak

(https://www.engeniusnetworks.eu/products/wireless/mesh-wifi/emr3000/#specification)

Sounds like there is no power LED, and the white LED should be implemented as wan LED or not at all. I also think it might be better to just refer to all LEDs as "status" LEDs:

```
	leds {
		compatible = "gpio-leds";

		status_white {
			label = "emr3000:white:status";
			gpios = <&gpio 18 GPIO_ACTIVE_HIGH>;
		};

		status_red {
			label = "emr3000:red:status";
			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
		};

		status_amber {
			label = "emr3000:amber:status";
			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
		};

		bluetooth {
			label = "emr3000:blue:bluetooth";
			gpios = <&gpio 19 GPIO_ACTIVE_HIGH>;
		};
	};
```

GPIOs for red vs. orange/amber might be wrong, as I cannot tell which is the red led from your data.

Reply:
Thanks for the comment.
Update LEDs node

Reply:
But still using the white LED for led-running creates the wrong impression, as with vendor firmware this would indicate a working internet/mesh connection.

Reply:
What's the vendor using for indicating booting and successful boot (i.e. running device)?